tests/bsim/bluetooth/audio/src/csip_set_member_test.c

```diff
@@ -105,6 +118,83 @@ static void test_main(void)
 	bt_conn_cb_register(&conn_callbacks);
 }
 
+static void test_disc(void)
+{
+	int err;
+
+	err = bt_enable(bt_ready);
+
+	if (err != 0) {
+		FAIL("Bluetooth init failed (err %d)\n", err);
+		return;
+	}
+
+	bt_conn_cb_register(&conn_callbacks);
+
+	PASS();
+}
+
+static void test_reconnect(void)
+{
+	int err;
+	uint8_t rsi[BT_CSIP_RSI_SIZE];
+
+	struct bt_data ad[] = {
+		BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+		BT_CSIP_DATA_RSI(rsi),
+	};
+
+	err = bt_enable(bt_ready);
+
+	if (err != 0) {
+		FAIL("Bluetooth init failed (err %d)\n", err);
+		return;
+	}
+
+	bt_conn_cb_register(&conn_callbacks);
+
+	WAIT_FOR_COND(disc);
+
+	err = bt_le_adv_start(BT_LE_ADV_CONN_NAME, ad, ARRAY_SIZE(ad), NULL, 0);
+	if (err != 0) {
+		FAIL("Advertising failed to start (err %d)\n", err);
+	}
+
+	PASS();
+}
+
+static void test_unranked(void)
+{
+	int err;
+
+	err = bt_enable(bt_ready);
+
+	if (err != 0) {
+		FAIL("Bluetooth init failed (err %d)\n", err);
+		return;
+	}
+
+	bt_conn_cb_register(&conn_callbacks);
+
+	PASS();
+}
+
+static void test_sirk(void)
```

Comment:
`test_sirk`, `test_unranked` and `test_disc` never start advertising, so how can they ever pass?

Reply:
`bt_enable(bt_ready)` has advertising start within.

Reply:
Ah, right, so it's in the callback - My bad :)